geometry_msgs/Pose[] pose_array
int64 r
int64 g
int64 b